dataflow/gemma/requirements.txt

```diff
@@ -0,0 +1,4 @@
+apache_beam[gcp]==2.54.0
```

Comment:
note that the samples repo has renovate bot configured that updates dependencies. we'd have to add a test to ensure that examples continue to work after dep upgrades or exclude this sample from renovatebot.

See https://github.com/GoogleCloudPlatform/python-docs-samples/pull/11252 for an example that has both of these suggestions.

Reply:
probably for the best to exclude it so we aren't having to continually download the model and run a GPU dataflow test, done